src/borg/archiver.py

```diff
@@ -2032,6 +2039,10 @@ def build_parser(self, prog=None):
                                    help='keep tag files of excluded caches/directories')
 
         archive_group = subparser.add_argument_group('Archive options')
+        archive_group.add_argument('--target', dest='target', metavar='ARCHIVE', default=None,
+                                   type=archivename_validator(),
+                                   help='place archive at <target>, don\'t replace existing archive '
```

Comment:
the word "place" here might suggest you can place it elsewhere (not in same repo).
also <target> is the option name, the value will show up as ARCHIVE.

maybe:
'create a new archive with the name ARCHIVE, do not replace existing archive'